saleor/graphql/discount/mutations.py

```diff
@@ -316,7 +328,12 @@ def perform_mutation(cls, _root, info, **data):
         voucher = cls.get_node_or_error(
             info, data.get("id"), only_type=Voucher, field="voucher_id"
         )
-        cls.add_catalogues_to_node(voucher, data.get("input"))
+        _input = data.get("input")
```

Comment:
Is there any reason for using underscore here?

Reply:
there is a build-in python `input`

Reply:
So we can different name instead i suppose, like `input_data`

Reply:
It's fine to override the builtins, we are not likely to ever use `input()` or `id()` in our code.

Reply:
Also, a leading underscore is used to mark unused variables.